package.json

```diff
@@ -22,11 +22,21 @@
     "url": "https://github.com/internetarchive/bookreader/issues"
   },
   "homepage": "https://github.com/internetarchive/bookreader#readme",
-  "dependencies": {},
   "private": false,
   "devDependencies": {
     "@babel/core": "7.12.9",
     "@babel/preset-env": "7.12.7",
+    "@internetarchive/ia-activity-indicator": "0.0.1",
+    "@internetarchive/ia-icons": "1.2.2",
+    "@internetarchive/ia-menu-slider": "0.1.10",
+    "@internetarchive/ia-sharing-options": "^0.1.2",
+    "@internetarchive/icon-bookmark": "1.1.3",
+    "@internetarchive/icon-collapse-sidebar": "^1.1.0",
+    "@internetarchive/icon-edit-pencil": "^1.1.5",
+    "@internetarchive/icon-magnify-minus": "^1.2.3",
+    "@internetarchive/icon-magnify-plus": "^1.2.3",
```

Comment:
We try to avoid `^` dependencies, so that all our environments are consistent (otherwise like if you npm install a month after me, you might get slightly different versions).

Reply:
Isn't that what `package-lock.json` is for?

Reply:
If I (very vaguely) recall correctly, `package-lock.json` isn't used for `npm i`? So running `npm i` will install from package.json using the appropriate version ranges. Running `npm ci` will install from package-lock.json, but also remove any existing node_modules (read: slow). The petabox repo uses npm shrinkwrap, which does something else which I can't remember :P Pinning them in the package.json seemed like the easiest approach, that gets us 90% of what we need without complicating things too much.

Reply:
(90% because there is still some wiggle room for secondary dependencies having different versions, but meh; super unlikely to cause in issue if a secondary dependency has like a patch update. I believe the npm-shrinkwrap approach does cover that last 10%, but also adds some complexity. Makes sense for a larger repo like petabox, but for BR its not worth it.)

Reply:
I looked into this and I think we should be using the `^` for these dependencies. When developers check out the the book reader repo, the `package-lock.json` file will guarantee that `npm install` installs the proper version, but still allows consumers to update transitive dependencies. 

By locking these versions down, we cannot install minor updates of these icons in petabox because BR is requiring these specific versions and we include these same icons in other dependencies. Locking things down in the `package.json` kind of breaks semantic versioning.

Here's a good article about the different purposes of the files: https://medium.com/@hossam.hilal0/package-json-vs-package-lock-json-vs-npm-shrinkwrap-json-33fcddc1521a

Reply:
Good article! I don't think we should do that as part of this PR (unless it blocks the migration), but seems good to me :+1:

Reply:
Sorry, what seems good to you?

Reply:
Switching to `^`

Reply:
resolving this one